Packs/Netskope/Integrations/integration-Netskope.yml

```diff
@@ -84,6 +84,12 @@ script:
         }
         var body;
         try {
+            // if fetching incidents, stringify long numbers to not round down long ids, e.g:
+            // "dlp_incident_id":3747385551915191779 --> "dlp_incident_id":"3747385551915191779"
+            if (stringifyData && stringifyData.toLowerCase() == 'true') {
+                stringifyBody = result.Body.replace(/([\[:])?(\d{11,})([,\}\]])/g, "$1\"$2\"$3");
+                return JSON.parse(stringifyBody).data
```

Comment:
Why do you return the data, and not the body itself like in the other return statement?
Also, I suggest having only one return statement, unless you feel there is a reason for more than one

Reply:
this is the return for the fetch incident only which stringify the results to handle the long numbers.
I know it is not the prettiest, but given the fact that we have no instance and every change requires a validation, I'm ok with this.